Source/Examples/ExampleLibrary/ExampleLibrary.csproj

```diff
@@ -1,27 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;net6.0;net6.0-windows</TargetFrameworks>
```

Comment:
Is there a benefit to having net6.0 and net6.0-windows?

Reply:
To support building examples on non-windows platforms, I assume.

Hard to remember my thought process from last November.

Reply:
I should be OK to just have `net6.0`, as the example library doesn't depend on any windows APIs. Publish produces separate packages for net6.0 and net6.0-windows so wouldn't be a problem, but would suggest dropping net6.0-windows where we don't need it.